src/chttpd/test/eunit/chttpd_open_revs_error_test.erl

```diff
@@ -50,7 +50,7 @@ create_doc(Url, Id) ->
 delete_db(Url) ->
     {ok, 200, _, _} = test_request:delete(Url, [?AUTH]).
 
-open_revs_error_test_() ->
+open_revs_error_test_disabled() ->
```

Comment:
Why not just delete these given that its a shards test?

Reply:
Good call will delete it. I started with wanting to minimize the diff sizes between master but deleted vs not is not that big of a deal I think